### PR TITLE
Add external db check on pr to ci pipeline

### PR DESCRIPTION
**Pull request by lucaschimweg**
Add a check for external DB functionality to pull request checks in the concourse pipeline (#168)

We could not test this on our concourse. Integrations and configurations for GCP are missing.

*Acceptance Steps**
Update Concourse Pipeline and see if new pull request steps are working

@kramerul

### CODE REVIEW

File: ci/pipelines/cf-for-k8s.yml [hunk 1]
```diff
@@ -1442,6 +1443,235 @@ jobs:
...
+        APP_REGISTRY_REPOSITORY_PREFIX: ((cf_for_k8s_private_dockerhub.username))
+        APP_REGISTRY_USERNAME: ((cf_for_k8s_private_dockerhub.username))
+        APP_REGISTRY_PASSWORD: ((cf_for_k8s_private_dockerhub.password))
+        USE_EXTERNAL_DB: "true"
+      file: cf-for-k8s-ci/ci/tasks/install-cf-on-gke/task.yml
+      input_mapping:
+        cf-for-k8s: cf-for-k8s-develop
+
+  - task: push-test-app
+    attempts: 3
```
Comment (jamespollard8) [inline ci/pipelines/cf-for-k8s.yml:1592]: please remove these, they're no longer desirable

Comment (lucaschimweg) [reply ci/pipelines/cf-for-k8s.yml:1592]: Which ones exactly?

**Comment by lucaschimweg** [comment]
Refactored, both `install-postgres` tasks use the same task definition now

File: ci/pipelines/cf-for-k8s.yml (hunk 1 above)
Comment (loewenstein) [reply ci/pipelines/cf-for-k8s.yml:1459]: I suppose @jamespollard8  is referring to the multiple 'attempts'?

File: ci/tasks/install-postgres/task.sh [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+EOT
+
+export POSTGRES_PASSWORD=$(kubectl get secret -n external-db postgresql -o jsonpath="{.data.postgresql-password}" | base64 -d)
+kubectl exec -n external-db statefulset/postgresql-postgresql -i -- psql "postgresql://postgres:$POSTGRES_PASSWORD@localhost:5432/postgres" < /tmp/setup_db.sql
+
+cat > db-metadata/db-values.yaml <<EOT
+#@data/values
+---
+capi:
+database:
```
Comment (kramerul) [inline ci/tasks/install-postgres/task.sh:56]: The indentation is not correct here.

File: ci/pipelines/cf-for-k8s.yml (hunk 1 above)
Comment (jamespollard8) [reply ci/pipelines/cf-for-k8s.yml]: Yes, the multiple attempts have been removed since the PR was initially opened. 👍

**Comment by ericpromislow** [comment]
We had to revert this commit because `set-pipeline` failed (which you can't run directly, unfortunately):
```
$ fly -t relint-ci set-pipeline -c ci/pipelines/cf-for-k8s.yml -p cf-for-k8s
error: yaml: unknown anchor 'on_failure_uptime_pr_comment' referenced
```

Between submission and merging, the `on_failure_uptime_pr_comment` anchor was replaced with the underlying inline code. I know, but the`context` field kept varying, so the anchored block was removed.

It would be easiest if you could create a new PR based on this one and that change. And this time we'll test it locally before pressing the big "Merge" button.

Thanks,
Eric

**Comment by kramerul** [comment]
Hi @ericpromislow,

I openend a new PR: #328

**Pull request merged**